.github/actions/runner_remove/action.yaml

```diff
@@ -0,0 +1,50 @@
+name: Delete runner VM
+description: Delete runner VM using create-vm.py
```

Comment:
> Delete runner VM using create-vm.py
Maybe you should consider naming the file the other way, or using two different scripts?